src/bundle/ui-dev/src/modules/universal-discovery/components/tab-selector/tab.selector.js

```diff
@@ -35,7 +43,14 @@ const TabSelector = () => {
                 });
 
                 return (
-                    <div className={className} key={tab.id} onClick={onClick} title={tab.label}>
+                    <div
+                        ref={selectorItemTooltipCallbackRef}
```

Comment:
Probably this would also work 🤔 
```suggestion
                        ref={ibexa.helpers.tooltips.parse}
```